index.html

```diff
@@ -35,6 +36,99 @@ <h2>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+    </div>
+    <div class="aboutme">
+        <h2>About me</h2>
+        <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <div>Get my resume</div>
+    </div>
+    <hr>
+    <div class="list">
+        <ul>
+            <li><img src="/Images/Rectangle55.png" alt="rectangle"> Language</li>
+            <li><p class="listsub">Javascript</p></li>
+            <li><p class="listsub">Ruby</p></li>
+            <li><p class="listsub">Html</p></li>
+            <li><p class="listsub">Css</p></li>
+            <li><img src="/Images/Rectangle56.png" alt="rectangle"> Frameworks</li>
+            <li><p class="listsub">Bootstrap</p></li>
+            <li><p class="listsub">Ruby on Rails</p></li>
+            <li><p class="listsub">RSPec</p></li>
+            <li><p class="listsub">CapyBara</p></li>
+            <li><p class="listsub">Selenium</p></li>
+            <li><img src="/Images/Circle.png" alt=""> Skills</li>
+            <li><p class="listsub">Codekit</p></li>
+            <li><p class="listsub">Githubt</p></li>
+            <li><p class="listsub">Codepen</p></li>
+            <li><p class="listsub">Gitlab</p></li>
+            <li><p class="listsub">Terminal</p></li>
+        </ul>
```

Comment:
Great use of a `ul` element 😎 Just a small suggestion 🧐 For better code organization and more control on each of the sections, you could have three `ul` tags separating Language, Frameworks, and Skills